src/registry/views/static/info.ts

```diff
@@ -7,7 +7,7 @@ oc.cmd.push(function() {
     window.location = thisComponentHref + $(this).val() + '/~info';
   });
 
-  $('.refresh-preview').click(function() {
+  refreshPreview = function() {
```

Comment:
I think you want to do either `var refreshPreview  = function() {` or `function refreshPreview() {` so it does not become a global variable.

Reply:
ok, fixed that to `var refreshPreview = ...`